apps/storybook/src/EventsCalendar.stories.tsx

```diff
@@ -1,16 +1,21 @@
-import { ComponentProps } from 'react';
 import { EventsCalendar } from '@asap-hub/react-components';
 import { createListCalendarResponse } from '@asap-hub/fixtures';
-import { number } from '@storybook/addon-knobs';
+import { boolean, number } from '@storybook/addon-knobs';
 
 export default {
   title: 'Templates / Events / Calendar',
 };
 
-const { items: calendars } = createListCalendarResponse();
-
-const props = (): ComponentProps<typeof EventsCalendar> => ({
-  calendars: calendars.slice(0, number('Calendar item', 1, { min: 0, max: 6 })),
-});
-
-export const Normal = () => <EventsCalendar {...props()} />;
+export const Normal = () => (
+  <EventsCalendar
+    calendars={
+      createListCalendarResponse(number('Calendar item', 1, { min: 0 }), {
+        activeGroups: boolean('Calendar connected to Group', true),
+        incompleteWorkingGroups: boolean(
```

Comment:
Should this be activeWorkingGroups to be consistent?

Reply:
Well it depends, the actual property is "complete" which means the working group is no longer active, I tried keeping consistency with how it's seen on Squidex